fdbserver/workloads/BackupS3BlobCorrectness.actor.cpp

```diff
@@ -587,6 +585,48 @@ struct BackupS3BlobCorrectnessWorkload : TestWorkload {
 				state Reference<IBackupContainer> lastBackupContainer =
 				    wait(BackupConfig(logUid).backupContainer().getD(cx.getReference()));
 
+				// Wait for backup to become restorable if it's still in progress
+				// This handles cases where cluster recoveries delay snapshot completion
```

Comment:
Can you make this a separate actor for clarity?

Reply:
np